components/threading/virtualized_thread_viewer/create_comment.tsx

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {memo, forwardRef} from 'react';
+
+import {UserProfile} from 'mattermost-redux/types/users';
+import {Post} from 'mattermost-redux/types/posts';
+import {$ID} from 'mattermost-redux/types/utilities';
+
+import GenericCreateComment from 'components/create_comment';
+import FormattedMarkdownMessage from 'components/formatted_markdown_message';
+import Constants from 'utils/constants';
+
+type Props = {
+    blockFocus: boolean;
+    channelId?: string;
+    channelIsArchived: boolean;
+    channelType: string;
+    isDeleted: boolean;
+    isFakeDeletedPost: boolean;
+    onHeightChange: () => void;
+    teammate?: UserProfile;
+    threadId: string;
+    latestPostId: $ID<Post>;
+};
+
+const CreateComment = forwardRef<HTMLDivElement, Props>(({
```

Comment:
It seemed easiest to do it this way. I could maybe refactor `RhsComment` and `RshRootPost` index files to receive post id instead of a post, but `CreateComment` has a bunch of extra html added so keeping the wrapper seems better to me.

Reply:
I guess we can keep that extra stuff in mind whenever we get around to cleaning that up. I'd like to combine RhsRootPost and RhsComment at some point too since they're mostly duplicate code. We used to style the root post differently so having two components kind of made sense then, but it's been a long time since that was the case.